### PR TITLE
fix: docs, subdomanin bug, name bug

### PR DESCRIPTION
**Pull request by AnguIar**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                      |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

File: src/downloads/models/downloadsManager.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ export class DownloadsManager {
         secretAccessKey: this.s3Config.secretAccessKey,
       },
       endpoint: this.s3Config.endpoint,
+      s3ForcePathStyle: this.s3Config.s3ForcePathStyle
     });
   }
```
Comment (cl-shaharshavit-rnd) [inline src/downloads/models/downloadsManager.ts:28]: if it works without adding the bucket to the key remove the comment

File: config/default.json [hunk 2]
```diff
@@ -15,6 +15,7 @@
     "accessKeyId": "minioadmin",
     "secretAccessKey": "minioadmin",
     "endpoint": "http://localhost:9000",
-    "bucket": "mapping-images"
+    "bucket": "mapping-images",
+    "s3ForcePathStyle": true
```
Comment (cl-shaharshavit-rnd) [inline config/default.json:19]: isn't s3 in the name redundant as it is in s3 object?

**Review by cl-shaharshavit-rnd — APPROVED** [review]

**Review by cl-shaharshavit-rnd — APPROVED** [review]

File: openapi3.yaml [hunk 3]
```diff
@@ -9,16 +9,18 @@ info:
...
       summary: download the requested file
       parameters:
-        - name: key
-          in: path
-          required: true
+      - name: key
+        in: path
+        required: true
+        schema:
+          type: string
```
Comment (asafMasa) [inline openapi3.yaml:23]: add 404

File: src/downloads/controllers/downloadsController.ts [hunk 4]
```diff
@@ -17,7 +17,9 @@ export class DownloadsController {
 
   public getResource: RequestHandler<{ key: string }> = (req, res, next) => {
     try {
-      const file = this.manager.download(req.params.key);
+      const fileName = req.params.key;
+      console.log(fileName);
```
Comment (asafMasa) [inline src/downloads/controllers/downloadsController.ts:21]: remove

**Review by asafMasa — CHANGES_REQUESTED** [review]
🏅

**Review by asafMasa — CHANGES_REQUESTED** [review]
fix prettier

**Review by asafMasa — APPROVED** [review]

**Pull request merged**